src/main/java/hinlok/parser/Parser.java

```diff
@@ -0,0 +1,73 @@
+package hinlok.parser;
+
+import hinlok.command.Command;
+import hinlok.command.ListCommand;
+import hinlok.command.ExitCommand;
+import hinlok.command.DeleteCommand;
+import hinlok.command.MarkCommand;
+import hinlok.command.UnmarkCommand;
+import hinlok.command.AddCommand;
+import hinlok.command.FindCommand;
```

Comment:
I like how you didn't use a wildcard import so that it's clear what you're importing